rust/unicode/src/lib.rs

```diff
@@ -37,8 +41,11 @@ pub fn linebreak_property(cp: char) -> u8 {
     }
 }
 
-// Return property, length
-// May panic if ix doesn't point to a valid character in the string
+/// The Unicode line breaking property of the given code point.
+///
+/// Look up the line breaking property for the first code point in the
+/// string. Return the property as a numeric value, and also the utf-8
+/// length of the codepoint, for convenience.
 pub fn linebreak_property_str(s: &str, ix: usize) -> (u8, usize) {
```

Comment:
If this is legitimately a public API I would return a struct with named fields. 🤷

Reply:
This is basically following the tradition of char_indices :) But I hear you.